cloud_build/stage.yaml

```diff
@@ -1,20 +1,16 @@
 steps:
   - name: gcr.io/cloud-builders/git
     args: ['submodule', 'update', '--init', '--recursive']
-  - name: gcr.io/cloud-builders/docker
-    entrypoint: '/bin/bash'
-    args:
-      - '-c'
-      - |-
-        set -e
-
-        echo "Building the website using a makefile..."
-        make build
   - name: gcr.io/flutter-dev-230821/firebase-ghcli
     entrypoint: '/bin/bash'
     args:
       - '-c'
       - |-
+        set -e
+        
+        npm install
+        npm run build
```

Comment:
What's the difference between prod vs build?

Reply:
A production build minifies HTML, CSS, etc. It also has a `robots.txt` to allow the site to be indexed, since we don't want staged sites to be indexed.

This will eventually be more clear once I can use the `dart_site` tooling here and use those commands, but Dart isn't available in these Cloud Build workflows currently. So I added `npm build` and `npm prod` to use temporarily.

Reply:
Thanks Parker :D